common-money.lic

```diff
@@ -64,11 +101,65 @@ module DRCM
   def check_wealth(currency)
     DRC.bput("wealth #{currency}", /\(\d+ copper #{currency}\)/i, /No #{currency}/i).scan(/\d+/).first.to_i
   end
-
   def wealth(hometown)
     check_wealth(hometown_currency(hometown))
   end
 
+  def get_total_wealth
+    # This method captures your current total on-hand wealth
+    # and returns a hash representing the numerical value in
+    # coppers of each currency.    
+
+    # Set up variables to capture each line of wealth output by currency
+    kronars_line = nil
+    lirums_line = nil
+    dokoras_line = nil
```

Comment:
Thanks for the quick updates, these lines can now be deleted since no longer being used
```suggestion
```

Reply:
> Thanks for the quick updates, these lines can now be deleted since no longer being used

Thanks for the suggestions. I'm working on deleting those now. I'm also getting an error, so diagnosing that.